public/app/features/explore/Logs/LogsContainer.tsx

```diff
@@ -59,45 +60,75 @@ interface LogsContainerProps extends PropsFromRedux {
 
 interface LogsContainerState {
   logDetailsFilterAvailable: boolean;
+  /**
+   * LogContext support for mixed data sources.
+   * @alpha
```

Comment:
Just a nit: not sure if this is needed? The interface doesn't seem exported anyways.

Reply:
100%, I think I did this more as a reflex than with a particupar purpose.